src/D2L.Security.OAuth2/Keys/Default/D2LSecurityToken.cs

```diff
@@ -46,7 +46,9 @@ internal sealed partial class D2LSecurityToken : SecurityKey {
 				this,
 				GetSignatureAlgorithm()
 			) {
-				CryptoProviderFactory = new D2LCryptoProviderFactory()
+				CryptoProviderFactory = new D2LCryptoProviderFactory {
+					CacheSignatureProviders = false

```

Comment:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Caching-in-Microsoft.IdentityModel
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1605